examples/samples/timeToLive.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  delay,
+  OnMessage,
+  OnError,
+  MessagingError,
+  ServiceBusMessage,
+  ReceiveMode,
+  Namespace,
+  SendableMessageInfo,
+  generateUuid
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const queuePath = process.env.QUEUE_NAME || "";
+const deadLetterQueuePath = queuePath + "/$DeadLetterQueue";
+const receiveClientTimeoutInMilliseconds = 10000;
+console.log("str: ", str);
+console.log("queue path: ", queuePath);
+console.log("deadletter queue path: ", deadLetterQueuePath);
+
+let ns: Namespace;
+
+/*
+  This sample demonstrates how TimeToLive property works. For more information, refer to
+  https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-expiration
+
+  On running this sample, you should immediately see a Pineapple Icecream message in the queue.
+  After 30 seconds, you should see the message moved to DLQ with appropriate reason being set.
+*/
+async function main(): Promise<void> {
+  try {
+    ns = Namespace.createFromConnectionString(str);
+    // Send messages with TTL property assigned
+    await SendMessagesAsync();
+
+    // Expire the messages
+    await delay(60000);
+
+    // Process expired messages from the Dead Letter Queue
+    await PickUpAndFixDeadletters();
+  } catch (err) {
+    console.log(">>>>> Error occurred in running sampple: ", err);
+  } finally {
+    console.log("\n>>>> Calling close....");
+    ns.close();
```

Comment:
Oh, the finally was wrapping over all of the code in main so I'm a little confused why it would miss errors thrown by any of the functions invoked from within it.
But yes, agree with dropping it for purpose of samples - have updated code in this and others..